include/cairo/context.hpp

```diff
@@ -3,6 +3,7 @@
 #include <cairo/cairo-xcb.h>
 
 #include <algorithm>
+#include <array>
```

Comment:
The missing `<array>` header is already in the hotfix branch, see #3038. Should also be on master within the week

Reply:
Alright!
Shall I update the PR to exclude this?

Reply:
Please remove them. My other PR adds them directly into `common.hpp`